bsp/esp-box/idf_component.yml

```diff
@@ -19,11 +19,11 @@ dependencies:
     override_path: "../../components/esp_lvgl_port"
 
   esp_codec_dev:
-    version: "^1.0.3"
+    version: "~1.1"
     public: true
 
   button:
-    version: ">=2.5"
+    version: "^2.5"
```

Comment:
Is it good to use this in `button` component? I think, better (safer) will be `~2.4`